.devcontainer/docker-compose.override.yaml

```diff
@@ -0,0 +1,43 @@
+name: lumigator
+
+services:
+
+  backend:
+    build:
+      context: .
+      dockerfile: "Dockerfile"
+    platform: linux/amd64
+    command: ["uv", "run", "uvicorn", "backend.main:app", "--host", "0.0.0.0", "--port", "8000",  "--reload"]
+    depends_on:
+      - localstack
+    ports:
+      - 80:8000
+    environment:
+      - DEPLOYMENT_TYPE=local
+      # The local file needs to be available through a mount,
+      # if persistence is needed
+      - SQLALCHEMY_DATABASE_URL=sqlite:///local.db
+      - S3_ENDPOINT_URL=http://localhost:4566
+      - AWS_ACCESS_KEY_ID=test
+      - AWS_SECRET_ACCESS_KEY=test
+      - AWS_DEFAULT_REGION=us-east-2
+      - AWS_ENDPOINT_URL=http://localhost:4566
+      - S3_BUCKET=lumigator-storage
+      - PYTHONPATH=/mzai/lumigator/python/mzai/backend
+      - PIP_REQS=/mzai/lumigator/python/mzai/evaluator/requirements.txt
+      - EVALUATOR_WORK_DIR=/mzai/lumigator/python/mzai/evaluator
+      - RAY_DASHBOARD_PORT=8265
+      - RAY_HEAD_NODE_HOST=ray
+      - MISTRAL_API_KEY=${MISTRAL_API_KEY}
+      - OPENAI_API_KEY=${OPENAI_API_KEY}
+      - RAY_WORKER_GPUS=0
+      - RAY_WORKER_GPUS_FRACTION=0
+    volumes:
+      - ./:/mzai
```

Comment:
If I'm not wrong, we are building with the Dockerfile located in the root of the repo, that one includes the whole mzai directory, so this volume is not needed. Am I right, or I'm missing something?

Reply:
Hi Mario! This volume is to mount the code in order to do local development. The docker-compose override is only used in the "local-up" (not start-lumigator), so this will be only used for local development.

Reply:
Yep, I already knew that, but I missed the "--reload" in the uv command, now everything makes sense 👍 Thank you!